alembic/versions/2025_03_06_3a0250e5526d_add_nallo_to_limitations_table.py

```diff
@@ -0,0 +1,54 @@
+"""add-nallo-to-limitations-table
+
+Revision ID: 3a0250e5526d
+Revises: 6f368f57df4a
+Create Date: 2025-03-06 11:07:29.158959
+
+"""
+
+from sqlalchemy.dialects import mysql
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "3a0250e5526d"
+down_revision = "6f368f57df4a"
+branch_labels = None
+depends_on = None
+
+
+new_workflow = "nallo"
+
+old_options = (
+    "balsamic",
+    "balsamic-pon",
+    "balsamic-qc",
+    "balsamic-umi",
+    "demultiplex",
+    "fastq",
+    "fluffy",
+    "microsalt",
+    "mip-dna",
```

Comment:
I would have to double check, but in some revisions I have seen that Jasen is already a workflow, maybe you need to include it here

Reply:
It is an option so you probably want to add it
<img width="1490" alt="Screenshot 2025-03-06 at 11 26 11" src="https://github.com/user-attachments/assets/1d1f7a2f-a6d3-4e7f-9ab7-094bfe4ca872" />

Reply:
done!